src/amo/components/SearchResult/index.js

```diff
@@ -59,10 +63,9 @@ export class SearchResultBase extends React.Component<InternalProps> {
     let imageURL = iconURL;
 
     if (addon && isTheme(addon.type)) {
-      // Since only newly created static themes will have more than one preview
-      // we will set up a fallback for now.
-      let themeURL =
-        getPreviewImage(addon, { index: 1 }) || getPreviewImage(addon);
+      const previewIndex = getPreviewIndexBySize(addon);
+
+      let themeURL = getPreviewImage(addon, { index: previewIndex });
```

Comment:
This will return `null` if `previewIndex` is undefined but I'd have to read the code to know that. To me, this says that you should be combining two methods into one. For example, instead of this:

```js
const previewIndex = getPreviewIndexBySize(addon);
let themeURL = getPreviewImage(addon, { index: previewIndex });
```

How about just a top-level call like...

```js
let themeURL = getPreviewImage(addon, { preferWidest: true });
```

Reply:
sure, I'll take another look. thanks for the feedback